nix-tools/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, symlinkJoin
```

Comment:
we can't reuse the default from nix-tools/default.nix here right?

Reply:
Why not? You can import Nix expressions from a `fetchTarball` just fine.

Reply:
It would require reworking the Nix expressions in nix-tools so that `haskell` can be provided as an optional argument. Which we could do after merging this PR.

Reply:
:+1: